### DISCUSSION TITLE
Trying to use rtx in vrchat

### DISCUSSION THREAD

**Discussion by ghost** [retroluxfilm/reshade-vrtoolkit#14] [General]
Main reason I wanted to use vrtoolkit was to use rtx in vr social games like vrc and neos, but since the high network usage reshade blocks the use of the depth map, any way for me to get past that/get the depth map any other way?

**Comment by retroluxfilm** [comment]
The depth map access is not supported in reshade for VR yet. See: https://github.com/crosire/reshade/pull/205

**Reply by ghost** [reply]
ah ok, well thank you.
